src/Elastic.Apm.AspNetCore/Elastic.Apm.AspNetCore.csproj

```diff
@@ -1,16 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
     <AssemblyName>Elastic.Apm.AspNetCore</AssemblyName>
     <RootNamespace>Elastic.Apm.AspNetCore</RootNamespace>
     <PackageId>Elastic.Apm.AspNetCore</PackageId>
     <Description>Elastic APM for ASP.NET Core. This package contains auto instrumentation for ASP.NET Core. See: https://github.com/elastic/apm-agent-dotnet/tree/master/docs</Description>
     <PackageTags>apm, monitoring, elastic, elasticapm, analytics, aspnetcore</PackageTags>
+    <TargetFrameworks>netstandard2.0;netcoreapp2.2</TargetFrameworks>

```

Comment:
🤔 this is not ok - we can't publish this as `netcoreapp2.2`.  I'll come back and think about this.

Reply:
> 🤔 this is not ok - we can't publish this as `netcoreapp2.2`. I'll come back and think about this.

What is the reason?

Reply:
> > 🤔 this is not ok - we can't publish this as `netcoreapp2.2`. I'll come back and think about this.
> 
> What is the reason?

Because we can't publish a single nuget package that has targets `netcoreapp2.2.` and `netstandard2.0` at the same time. The `dotnet publish` command also fails with this `.csproj`:

```
/local/share/dotnet/sdk/3.1.100/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.CrossTargeting.targets(27,5): error : The 'Publish' target is not supported without specifying a target framework. The current project targets multiple frameworks, please specify the framework for the published application. 
```

 which is not the case when we publish the `Elastic.Apm` package, which targets `netstandard2.0` and `net461`. 

The fact that `dotnet publish` fails without specifying a target kind of makes sense if you think about it: let's say you have an app that targets `netcoreapp2.2.`, or `netcoreapp3.0` - if you only add this to its `.csproj` file:

```
 <PackageReference Include="Elastic.Apm.AspNetCore" ...>
```

and `Elastic.Apm.AspNetCore` has both `netcoreapp` and `netstandard` target, then what target (meaning what `.dll` file in the nuget pacakge) should the `netcoreapp2.2` or `netcoreapp3.0` app pick? Both could be good, both are supported.

Overall I think the default approach should be to target `netstandardx` if we are a library.

But.. I think we don't even need `netcoreapp2.0`, you eliminated the need here: https://github.com/elastic/apm-agent-dotnet/pull/632/commits/7a09246f72adb8d1b9a8fa3b798a9ffa33e2308b , right?

Reply:
Hm, interesting. I don't remember case when I need to target both netstandard and netcoreapp in nuget package, and your explanation sounds reasonable.

However, why are you using `publish` instead of `pack`? Also, I thought that package referencing works in a little bit different manner, at least if we have two different netstandard targets, e.g. `netstandard1.6` and `netstandard2.0`, the latest possible target will be used. In case of `netcoreapp2.2` it will be `netstandard2.0`. Also, such targets can be packed inside one nuget package.

Reply:
> However, why did you use publish instead of pack?

Yeah, you are right - and I have no idea why I tested with `dotnet publish` - sorry, that was a quick investigation on a train 🚋 ... And yes, both `netstandard2.0` and `netcoreapp2.2` can be in the same package, so that statement was definitely not true.

Nevertheless:

> BTW, it seems you're right and we don't need netcoreapp2.2 target.

Yeah, I think we both agree on this one, so, although the discussion is interesting, ultimately I guess we can just remove `netcoreapp2.2`.